Doc/c-api/typeobj.rst

```diff
@@ -1368,6 +1371,23 @@ and :c:data:`PyType_Type` effectively act as defaults.)
    debugging aid you may want to visit it anyway just so the :mod:`gc` module's
    :func:`~gc.get_referents` function will include it.
 
+   Heap types (:c:macro:`Py_TPFLAGS_HEAPTYPE`) must visit their type with::
```

Comment:
This doesn't seem relevant `PyObject_VisitManagedDict`, but no harm in adding it.

Reply:
This doc was forgotten since Python 3.8, it requires to fix an important bug.